include/accessor/accessor.hpp

```diff
@@ -32,19 +32,19 @@ namespace accessor
       using type = R (C::*)(Args...) const;
     };
 
-    template<class Tag, class T>
```

Comment:
This parameter is needed due to its uniqueness - that's why  there is `Tag` and `T` - here is a description https://github.com/insooth/insooth.github.io/blob/master/accessing-private-members.md#uniqueness

Reply:
I read it but what you have in the code it's not what you describe in the article. In the code the parameters are always the same, they're always equal, so I can't see how uniqueness can be applied currently. In addition after the modification all tests are executed without any problem.